src/lang/pl.json

```diff
@@ -513,6 +513,7 @@
   "FIELD_ENABLED": "włączone",
   "FIELD_ADD_FRAME": "Dodaj klatkę",
   "FIELD_CLONE_FRAME": "Klonuj klatkę",
+  "FIELD_CLONE_FRAMES": "Klonuj klatki"
```

Comment:
you missed adding a `,` at the end of this line, not a problem though the test suite picked up the issue and I've fixed the file :-)

Reply:
A, my bad sorry... I will look out next time, thanks! :-)